src/main/resources/META-INF/rewrite/spring-boot-20.yml

```diff
@@ -118,7 +118,7 @@ recipeList:
 
 ---
 type: specs.openrewrite.org/v1beta/recipe
-name: org.openrewrite.java.spring.boot2.MigrateHibernateContraintsToJavax
+name: org.openrewrite.java.spring.boot2.MigrateHibernateConstraintsToJavax
```

Comment:
This is probably problematic, but it'd be nice if it happened.

Reply:
I'm going to agree with you here; since this is highly likely limited to this repository I think it's fine to change. There's a slim chance some of our users refer to this one directly, but even then the change should be clear.